lib/systems/include/Controllers/SimpleController.h

```diff
@@ -28,8 +28,12 @@ class TorqueControllerSimple : public Controller
         writeout_.command = BaseControllerParams::TC_COMMAND_NO_TORQUE;
         writeout_.ready = true;
     }
-
+    /// @brief calculates torqe output based off max torque and simple torque scaling metric defined in constructor
+    /// @param PedalsSystemData_s &pedalsData 
     void tick(const PedalsSystemData_s &pedalsData);
+    /// @brief ticks controller
+    /// @param SharedCarState_s &state 
+    /// @return TorqueControllerOutput_s
```

Comment:
Again, if this is identical to a parent class's documentation, then simply refer to the parent class.